CHANGELOG.md

```diff
@@ -1,17 +1,24 @@
 # CHANGELOG
 
+## v0.14.0
+
+*Aug 9, 2024*
+
+This release reinstates boltdb and cleveldb as deprecated backend types.
+Please not that we discourage using them, as we plan to drop them in a future release.
+
+### DEPENDENCIES
+
+- reinstate BoltDB and ClevelDB as backend DBs
+  ([\#177](https://github.com/cometbft/cometbft-db/pull/177))
+
 ## v0.13.0
 
 *Aug 2, 2024*
 
 This release:
-- changes the contract of the Iterator Key() and Value() APIs: the caller
-is now responsible for creating a copy of their returned value if they want to modify it.
-- bumps the Go version to 1.22.5 (from 1.22). Go 1.22.5 does not introduce new language
```

Comment:
we did bump go to 1.22.5 right ? if so, we need to keep this here

Reply:
yes we did